### DISCUSSION TITLE
How to ensure whole subject image, Flipbook toolbar is in view

### DISCUSSION THREAD

**Discussion by seanmiller26** [zooniverse/front-end-monorepo#6210] [Feature Requests]
## Package
_Choose from the list:_
- [ ] app-content-pages
- [ ] app-project
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [ ] lib-react-components
- [ ] unknown

## Is your feature request related to a problem? Please describe.

This is mostly related to survey tasks with a flipbook tool, however may apply to video subjects as well. 

On a larger screen, the subject is allowed to take a large portion of the screen, however this pushes the flipbook toolbar below the fold, which includes the play button. So, a volunteer must scroll down to initiate playback.

<img width="1000" alt="Screenshot 2024-04-26 at 1 56 22 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/89043489/6a4b3003-1cc3-467b-bfc4-1e9b09579aea">
<img width="1000" alt="Screenshot 2024-04-26 at 1 56 30 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/89043489/d6c8ef72-93ef-406b-9246-2f5ec580541c">

Here it is with the 'Limit subject image height option' enabled: (Note this detail) - The max height will be the image's original pixel height.

<img width="1000" alt="Screenshot 2024-04-26 at 5 05 38 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/89043489/38df1f40-b2b5-4fda-adeb-3dab93da8caa">

## Describe the solution you'd like
We should aim for some middle ground between the current implementation and the limit height option. I believe this would be best solved through responsive settings rather than another lab option.

Could we implement a VH restriction on the subject image? What about ensuring we always show the flipbook toolbar within the viewfinder? Any other options?

## Additional context
We would not want this to affect other subject viewers ie. transcription, where we want the image as large as possible.

**Comment by eatyourgreens** [comment]
In a related bug, when limited height is enabled (it's a project builder setting), the subject is tiny on mobile. See #6061. 

<img width=400 alt="Squirrel Mapper on my phone, with a subject image that's only about half the width of the screen." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/4bc642f3-c6b7-4d9e-828d-dde4f206f58a" >

**Comment by seanmiller26** [comment]
This may be at least partially solved if we address the width of the task area. Right now it is a constant size, and quite narrow. If we allow it to be wider it would not only give the survey selection options more room, but would also make the subjects smaller.
